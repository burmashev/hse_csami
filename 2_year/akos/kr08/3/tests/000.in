./blockwise_hash 42 4100 2048
