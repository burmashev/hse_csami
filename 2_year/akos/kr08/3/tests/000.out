[1]\tthis_block_is_good
[2]\tthis_block_is_bad
[3]\tthis_block_is_ugly
